Packs/PingIdentity/Integrations/PingOne/PingOne.yml

```
@@ -0,0 +1,228 @@
+      required: true
+      default: true
+      description: A username to activate.
+    description: Activate a user's account.
+  - name: pingone-set-password
+    arguments:
+    - name: username
+      required: true
+      default: true
+      description: The username to change the password for.
+    - name: password
+      required: true
+      description: The new password.
+    description: Sets a user's password.
+  - name: pingone-add-to-group
+    arguments:
+    - name: username
+      description: 'Username '
+    - name: groupName
+      description: Group Name
+    - name: groupId
+      description: Group ID
+    - name: userId
+      description: User ID
+    description: Add user to the group.
+  - name: pingone-remove-from-group
+    arguments:
+    - name: username
+      description: Username
+    - name: groupName
+      description: Group Name
+    - name: userId
+      description: User ID
+    - name: groupId
+      description: Group ID
+    description: Remove user from the group.
+  - name: pingone-get-groups
+    arguments:
+    - name: username
+      description: Username
+    outputs:
+    - contextPath: PingOne.Account.Group
+      description: PingOne groups of the user.
+      type: string
+    - contextPath: PingOne.Account.ID
+      description: PingOne account ID.
+      type: string
+    - contextPath: PingOne.Account.Type
+      description: PingOne account type.
+      type: string
+    - contextPath: PingOne.Account.Group.ID
+      description: Group unique ID.
+      type: string
+    - contextPath: PingOne.Account.Group.Name
+      description: Group name.
+      type: string
+    description: Returns user's group memberships.
+  - name: pingone-get-user
+    arguments:
+    - name: username
+      description: Username
+    - name: userId
+      description: User ID
+    - name: detailed
+      description: Return PingOne User details.
+    outputs:
+    - contextPath: PingOne.Account.ID
+      description: PingOne account ID.
+      type: string
+    - contextPath: PingOne.Account.Username
+      description: PingOne account username.
+      type: string
+    - contextPath: PingOne.Account.DisplayName
+      description: PingOne account display name.
+      type: string
+    - contextPath: PingOne.Account.Email
+      description: PingOne account email.
+      type: string
+    - contextPath: PingOne.Account.Enabled
+      description: PingOne account enabled status.
+      type: boolean
+    - contextPath: Account.CreatedAt
+      description: PingOne account create date.
+      type: date
+    - contextPath: PingOne.Account.UpdatedAt
+      description: PingOne account updated date.
+      type: date
+    description: Returns a PingOne user.
+  - name: pingone-create-user
+    arguments:
+    - name: username
+      required: true
+      description: The username to create.
+    - name: populationId
+      description: PingOne population ID where the new user will be created
```

Shouldn't it be mandatory arg?
```suggestion
    - name: populationId
      description: PingOne population ID where the new user will be created.
      required: true
```